webpack/assets/javascripts/react_app/routes/Statistics/StatisticsPage/Statistics/Statistics.js

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withRenderHandler } from '../../../../common/HOC';
 import StatisticsChartsList from '../../../../components/StatisticsChartsList';
+import { translate as __ } from '../../../../common/I18n';
 
-const Statistics = ({ statisticsMeta }) => (
-  <StatisticsChartsList data={statisticsMeta} />
-);
+const Statistics = ({ statisticsMeta }) => {
+  if (!statisticsMeta) {
+    return <p>{__('No Charts Loaded')}</p>;
```

Comment:
does it make sense to use EmptyState component?

Reply:
Yes but we should provide info in it and I'm not sure what info can we provide on "why aren't there any charts?"